docs/release_notes/7.0.14-8.md

```diff
@@ -0,0 +1,49 @@
+# Percona Server for MongoDB 7.0.14-8 ({{date.7_0_14}})
+
+[Installation](../install/index.md){.md-button}
+
+Percona Server for MongoDB 7.0.14-8 is an enhanced, source-available, and highly-scalable database that is a
+fully-compatible, drop-in replacement for MongoDB Community Edition 7.0.12.
+
+Percona Server for MongoDB 7.0.14-8 includes the improvements and bug fixes of [MongoDB 7.0.14 Community Edition](https://www.mongodb.com/docs/manual/release-notes/7.0/#7.0.14---aug-26--2024) and supports its protocols and drivers.
+
+## Release Highlights
+
+This release of Percona Server for MongoDB includes the following features and improvements:
+
+### Enhanced Telemetry for better product usage reporting
```

Comment:
it was already last release? No?
https://docs.percona.com/percona-server-for-mongodb/7.0/release_notes/7.0.12-7.html